manifests/CarlHetherington/DCP-o-matic/2.14.33.yaml

```
@@ -0,0 +1,17 @@
+Id: CarlHetherington.DCP-o-matic
+Version: 2.14.33
+Name: DCP-o-matic
+Publisher: CarlHetherington
+License: GNU GPLv2
+AppMoniker: dcpomatic
+Tags: DCP, movie
+Description: DCP-o-matic is a free, open-source program to create Digital Cinema Packages (DCPs) from videos, images, sound and subtitle files.
+Homepage: https://dcpomatic.com/
+Installers:
+  - Arch: x64
+    Url: https://dcpomatic.com/dl.php?id=windows-64&version=2.14.33&paid=0
```

```suggestion
    Url: https://dcpomatic.com/dl.php?id=windows-64&version=2.14.33
```
https://dcpomatic.com/dl.php?id=windows-64&version=2.14.33&paid=0
redirects to
`https://dcpomatic.com/dl.php?id=windows-64&version=2.14.33`